streamalert_cli/lookup_tables/handler.py

```diff
@@ -80,9 +81,85 @@ def _subcommands(cls):
             'get': LookupTablesGetKeySubCommand,
             'set': LookupTablesSetSubCommand,
             'list-add': LookupTablesListAddSubCommand,
+            'set-from-json-file': LookupTablesSetFromFile,
         }
 
 
+class LookupTablesSetFromFile(CLICommand):
+
+    @classmethod
+    def setup_subparser(cls, subparser):
+        set_parser = generate_subparser(
+            subparser,
+            'set-from-json-file',
+            description='Pushes the contents of a given json file into the LookupTable key',
+            subcommand=True
+        )
+
+        set_parser_epilog(
+            set_parser,
+            epilog=(
+                '''\
+                Examples:
+
+                    manage.py lookup-tables set-from-json-file -t [table] -k [key] -f \
+[path/to/file.json]
+                '''
+            )
+        )
+
+        set_parser.add_argument(
+            '-t',
+            '--table',
+            help='Name of the LookupTable',
+            required=True
+        )
+
+        set_parser.add_argument(
+            '-k',
+            '--key',
+            help='Key to modify on the LookupTable',
+            required=True
+        )
+
+        set_parser.add_argument(
+            '-f',
+            '--file',
+            help='Path to the json file, relative to the current working directory',
+            required=True
+        )
+
+    @classmethod
+    def handler(cls, options, config):
+        print('==== LookupTables; Set from JSON File ====')
+
+        table_name = options.table
+        key = options.key
+        file = options.file
```

Comment:
don't use `file` as a var name if you can avoid it. also, I don't think you need to re-ref these from the `options` named tuple..

Reply:
Sorry; force of habit. I was de-refing them to be explicit about what keys are expected from `options`, but I realize that the `setup_subparser` does exactly that but better; I will remove